### PR TITLE
Misc fixes 2

### PR DESCRIPTION
**Pull request by mschwoer**
- unify import of njit
- fix typo in config (breaking ;-))

### CODE REVIEW

File: alphadia/constants/default.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ library_prediction:
 
 # define custom alphabase modifications not part of unimod or alphabase
 # also used for decoy channels
-custom_modififcations:
```
Comment (GeorgWa) [inline alphadia/constants/default.yaml:56]: It’s breaking in theory :D alphabase can’t use the custom mods with multiprocessing yet. So it’s not being used (yet)

File: alphadia/quadrupole.py [hunk 2]
```diff
@@ -13,7 +6,7 @@
 from alphadia import utils
 
 
-@alphatims.utils.njit
```
Comment (GeorgWa) [inline alphadia/quadrupole.py:16]: Did you check the alphatims function? I wonder why I did this in the first place ( I already see some weird numbs thing happening :D )

**Review by GeorgWa — APPROVED** [review]
LGTM

File: alphadia/quadrupole.py (hunk 2 above)
Comment (mschwoer) [reply alphadia/quadrupole.py:16]: this is the `@alphatims.utils.njit` decorator
```
def njit(_func=None, *args, **kwargs):
    import numba
    return numba.njit(_func, *args, **kwargs)
```

Comment (GeorgWa) [reply alphadia/quadrupole.py:16]: So much value

**Pull request merged**